src/__test__/cart.test.ts

```diff
@@ -625,4 +625,4 @@ describe('Cart| Order  management for guest/buyer', () => {
       expect(response.body.data.cart).toBeDefined;
     });
   });
-});
+});
```

Comment:
@Ndevu12 , @GSinseswa721 , please check the 'Order Management Tests' suite starting from line 392 to see if they are written correctly. for ex, test on line 398, 436 (their test names).